.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,97 @@
+name: Bug report [Recommended Template]
+description: Something unexpected happen? Report an error, crash, or an incorrect behavior here.
+title: "[Bug]: <your-title>"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening an issue. **These GitHub issues** are only for community reporting and assistance; as such, we don't have a guaranteed SLA.
+        
+
+        **If you have an active MongoDB Atlas Support contract**, the best way for us to assist you with the AWS CDK MongoDB Atlas is through a support ticket.
+
+        **Please note:** In order for us to provide the best experience in supporting our customers, we kindly ask to make sure that all the following sections are correctly filled with all the required information. 
+        Our support will prioritise issues that contain all the required information that follows the [one-click reproducible issues principle](../../README.md#one-click-reproducible-issues-principle).
+  
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered. If the issue has been reported add a reaction, i.e. +1, to the issue.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  
+  - type: input
+    id: package_version
+    attributes:
+      label: CDK package version
+      description: What CDK Atlas package version are you running?
+      placeholder: "Example: v2.0.0"
+    validations:
+      required: true
+
+  - type: input
+    id: cfn_resource_version
+    attributes:
+      label: CFN Resource version
+      description: What CFN resource version are you using in your AWS account?
+      placeholder: "Example: v1.6.6"
+    validations:
+      required: true
+
+  - type: input
+    id: cfn_resource_region
+    attributes:
+      label: CFN Resource Region
+      description: In which AWS Region are you running your CFN stack?
+      placeholder: "US-EAST-1"
```

Comment:
in AWS terminology they use lowercase, e.g. us-east-1

Reply:
updated